applications/TrilinosApplication/tests/test_TrilinosApplication_mpi.py

```diff
@@ -35,7 +35,6 @@ def AssembleTestSuites():
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([test_trilinos_linear_solvers.TestLinearSolvers]))
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([test_trilinos_matrix.TestTrilinosMatrix]))
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([test_trilinos_redistance.TestTrilinosRedistance]))
-    smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([test_trilinos_redistance.TestTrilinosRedistanceInMemory]))
```

Comment:
why do you remove this test?
IIRC this was intended to check the partitioner

however I agree that this is not the place to put such a test :D

Seeing that you changed the test to use the in memory partitioner I am ok with it

(also a student of mine is currently writing some tests for the partitioner itself :) )

FYI @roigcarlo

Reply:
I already discussed this with @roigcarlo. It seems that this was a leftover from when he was implementing it. We agreed on keeping the partition in memory in here and remove the standard one (which is extensively tested). Also, note that the partition in memory has its own test.